Build technique for both Linux and macOS?

I note that there's no documented way to build for macOS (and the homebrew version 1.7.1 doesn't seem to work any more).

I also note that this doesn't build as suggested in the readme on Linux (./configure)

Building hints appreciated.

Ah, I see for Linux, the first thing is the instructions  in the README are missing the need to `./autogen.sh`, which is what creates the ./configure script.  (Sorry, it's been years since I've used automake and friends.)

In my case though (Ubuntu 18.04) the configure fails with `configure: error: OpenSSL 1.1.0 or later is required.` although `apt-get install openssl` gets `openssl is already the newest version (1.1.1c-1+ubuntu18.04.1+deb.sury.org+1)`

--- a/README.md
+++ b/README.md
@@ -27,7 +27,7 @@ removal and extraction.
 
 ## BUILDING
 
-Ths build technique works on Linux and macOS, if you have the necessary tools installed:
+This build technique works on Linux and macOS, if you have the necessary tools installed:
 ```
   ./autogen.sh
   ./configure
@@ -36,11 +36,10 @@ Ths build technique works on Linux and macOS, if you have the necessary tools in
 ```
 
 * On Linux, (tested on Debian/Ubuntu) you may need `sudo apt-get update && sudo apt-get install build-essential autoconf libtool libssl-dev python3-pkgconfig libcurl4-gnutls-dev`
-* On macOS with Homebrew, you probably need to do these tings before autogen.sh and configure:
+* On macOS with Homebrew, you probably need to do these things before autogen.sh and configure:
 ```
   brew install openssl@1.1 automake pkg-config libtool
-  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
-  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
+  export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
 ```
 
 ## USAGE

--- a/README.md
+++ b/README.md
@@ -25,13 +25,22 @@ valid signature for a CAB file containing Java files. It supports getting
 the timestamp through a proxy as well. It also supports signature verification,
 removal and extraction.
 
-## INSTALLATION
+## BUILDING
 
-The usual way:
+Ths build technique works on Linux and macOS, if you have the necessary tools installed:
 ```
+  ./autogen.sh
   ./configure
   make
   make install
+```
+
+* On Linux, (tested on Debian/Ubuntu) you may need `sudo apt-get update && sudo apt-get install build-essential autoconf libtool libssl-dev python3-pkgconfig libcurl4-gnutls-dev`
+* On macOS with Homebrew, you probably need to do these tings before autogen.sh and configure:
+```
+  brew install openssl@1.1 automake pkg-config libtool
+  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
+  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
 ```
 
 ## USAGE

